package.json

```diff
@@ -2,7 +2,7 @@
   "name": "serverless-mfa-api",
   "description": "A Serverless API for registering and validating Multi-Factor Authentication methods. Currently supports Time-based One Time Passwords (TOTP) and FIDO U2F devices (YubiKeys).",
```

Comment:
I looked for a version bump, but there doesn't seem to be a version number. Is that because this isn't a library?

Reply:
Yep, you got it